scripts/core/directives/FiletypeIconDirective.ts

```diff
@@ -34,7 +36,7 @@ export default angular.module('superdesk.core.directives.filetypeIcon', [])
                     return;
                 }
 
-                element.attr('title', `${gettextCatalog.getString('Article Type')}: ${item.type}`);
+                element.attr('title', `${gettext('Article Type')}: ${item.type}`);
```

Comment:
Translate entire phrase.

Reply:
Done.
I think also type should be translated.